sklearn/decomposition/dict_learning.py

```diff
@@ -56,18 +58,22 @@ def _sparse_encode(X, dictionary, gram, cov=None, algorithm='lasso_lars',
         omp: uses orthogonal matching pursuit to estimate the sparse solution
         threshold: squashes to zero all coefficients less than regularization
         from the projection dictionary * data'
+        elastic_net_cd: uses the coordinate descent method to compute the
+        Elastic Net solution (linear_model.ElasticNet).
 
-    regularization : int | float
+    regularization : int | float | float, float
         The regularization parameter. It corresponds to alpha when
-        algorithm is 'lasso_lars', 'lasso_cd' or 'threshold'.
-        Otherwise it corresponds to n_nonzero_coefs.
+        algorithm is 'lasso_lars', 'lasso_cd', 'threshold', or
+        'elastic_net_cd'. With 'elastic_net_cd', it corresponds to alpha and
+        the l1_ratio. Otherwise it corresponds to n_nonzero_coefs.
```

Comment:
I think it would be more conventional to include a separate l1_ratio parameter...?